src/content/platform-integration/ios/setup-ios.md

```diff
@@ -0,0 +1,270 @@
+---
+title: Set up iOS development
+description: >-
+  Configure your development environment to
+  run, build, and deploy Flutter apps for iOS devices.
+short-title: Set up iOS
+---
+
+Learn how to set up your development environment
+to run, build, and deploy Flutter apps for iOS devices.
+
+:::note
+If you haven't set up Flutter already,
+visit and follow the [Get started with Flutter][] guide first.
+
+If you have already installed Flutter,
+ensure that it's [up to date][].
+:::
+
+[Get started with Flutter]: /get-started/install
+[up to date]: /install/upgrade
+
+[up to date]: /install/upgrade
+
+## Install Xcode
+
+With Xcode, you can run Flutter apps on
+an iOS physical device or on the iOS Simulator.
+
+If you haven't done so already,
+install and set up [Xcode][].
+If you have installed Xcode, ensure that
+it's up to date and continue to the next step.
+(After setting up the command-line tools
+and accepting the license, as shown here.)
+
+[Xcode]: https://developer.apple.com/xcode/
+
+To configure Xcode's command-line tools,
+use the following command:
+
+```console
+$ sudo sh -c 'xcode-select -s /Applications/Xcode.app/Contents/Developer && xcodebuild -runFirstLaunch'
+```
+
+Sign the Xcode license agreement:
+
+```console
+$ sudo xcodebuild -license
+```
+
+## Configure your target iOS device
+
+Note that the behavior of the iOS Simulator can vary
+from an actual physical device. It's easier to get
+the Simulator up and running than a physical device,
+which requires more setup.
+For that reason, we recommend starting with the Simulator.
+
+{% tabs "ios-simulator-or-physical-device" %}
+{% tab "Simulator" %}
+
+### Configure your iOS Simulator
```

Comment:
I have never needed to do a single step in the "Configure your iOS Simulator" section. It was added in https://github.com/flutter/website/pull/9054 which links to https://github.com/flutter/website/issues/8990, which seems unrelated...
@vashworth any idea why we would need to keep the "use a 64-bit device" or "high-screen density" parts?

Current page is:
https://docs.flutter.dev/get-started/install/macos/mobile-ios#configure-your-ios-simulator

This PR is staged at https://flutter-docs-prod--pr11963-get-st-9rrn1yhn.web.app/platform-integration/ios/setup-ios